app/services/nomis/elite2/movement_api.rb

```diff
@@ -26,6 +26,15 @@ def self.movements_for(offender_no)
           api_deserialiser.deserialise(Nomis::Movement, movement)
         }
       end
+
+      def self.latest_movement_for(offender_no)
+        route = '/elite2api/api/movements/offenders?movementTypes=ADM&movementTypes=TRN&movementTypes=REL'
```

Comment:
Seems a shame to have the same code as the other function, might be worth replacing with something like ...

```suggestion
    movements_for(offender_no)&.first
```